integrationtests/src/test/java/io/kroxylicious/proxy/schema/validation/JsonSyntaxValidationIT.java

```diff
@@ -53,8 +52,8 @@ public class JsonSyntaxValidationIT extends BaseIT {
 
     @Test
     public void testInvalidJsonProduceRejected(KafkaCluster cluster, Admin admin) throws Exception {
-        assertEquals(1, cluster.getNumOfBrokers());
-        createTopics(admin, List.of(new NewTopic(TOPIC_1, 1, (short) 1)));
+        assertThat(cluster.getNumOfBrokers()).isEqualTo(1);
```

Comment:
Could use 
```suggestion
        assertThat(cluster.getNumOfBrokers()).isOne();
```

Not sure if it really makes any difference.

Reply:
Excellent point. Have changed this in all relevant places now.